web/src/components/RunDetailLayout/RunDetailLayout.tsx

```diff
@@ -42,29 +43,31 @@ const RunDetailLayout = ({test: {id, name, trigger, version = 1}, test}: IProps)
   );
 
   return (
-    <S.Container>
-      <Tabs
-        activeKey={mode}
-        centered
-        onChange={activeKey => {
-          TestRunAnalyticsService.onChangeMode(activeKey as RunDetailModes);
-          navigate(`/test/${id}/run/${run.id}/${activeKey}`);
-        }}
-        renderTabBar={renderTabBar}
-        tabBarExtraContent={tabBarExtraContent}
-        destroyInactiveTabPane
-      >
-        <Tabs.TabPane tab="Trigger" key={RunDetailModes.TRIGGER}>
-          <RunDetailTrigger test={test} run={run} isError={isError} />
-        </Tabs.TabPane>
-        <Tabs.TabPane tab="Trace" key={RunDetailModes.TRACE}>
-          <RunDetailTrace run={run} testId={id} />
-        </Tabs.TabPane>
-        <Tabs.TabPane tab="Test" key={RunDetailModes.TEST}>
-          <RunDetailTest run={run} testId={id} />
-        </Tabs.TabPane>
-      </Tabs>
-    </S.Container>
+    <TestOutputProvider testId={id} runId={run.id}>
```

Comment:
It seems that we have all the providers for the TestRun page here `/src/pages/RunDetail/RunDetail.tsx`. Any reason why not having the same pattern for this provider?

Reply:
The only reasoning was to avoid having to render it before we actually need it, I'll move it up just so it matches with the rest of the stuff